config/hydroshare-config.yaml

```diff
@@ -1,7 +1,7 @@
 ### Local Configuration Variables ###
 HS_PATH: /home/hydro/hydroshare
 HOST_SSL_DIR: /home/hydro/hs-certs
-HS_DATABASE: pg.development.sql
+HS_DATABASE: pg.www.sql
```

Comment:
This file is in `.gitignore` and changes to it should not be checked in. Please revert them.